### DISCUSSION TITLE
Direction

### DISCUSSION THREAD

**Discussion by LoveMyKeys** [PlummersSoftwareLLC/NightDriverStrip#604] [Q&A] [answered]
Can someone point me in the right direction with bringing up mesmerizer?

mesmerizer, wrover 16mb/8mb, pinout as default. Website is up, able to connect to it so the board is working.

![image](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/154417166/01e49819-3158-4be4-b8ec-82f22f6c23e6)

no clue what might be the problem, maybe someone has seen it before?

thanks in advance,

LMK

**Comment by LoveMyKeys** [comment]
Oh, this is shown both on website flash and my own build flash.

**Comment by robertlipe** [comment]
I'd find one of the many schematic/timing level explanations of HUB75 and
put a scope and/or analyzer on thpins and look for one or more that are
miswired or stuck.

Oh, and Remember that ESP32-S3 won't work....

Good luck.

**Reply by LoveMyKeys** [reply]
Thanks Robert.

**Comment by LoveMyKeys** [comment, marked as answer]
I ordered a different panel and that worked instant.
